cvat-ui/src/containers/actions-menu/actions-menu.tsx

```diff
@@ -125,11 +121,8 @@ function ActionsMenuContainer(props: OwnProps & StateToProps & DispatchToProps):
     } = props;
 
 
-    const loaders = annotationFormats
-        .map((format: any): any[] => format.loaders).flat();
-
-    const dumpers = annotationFormats
-        .map((format: any): any[] => format.dumpers).flat();
+    const loaders = annotationFormats.loaders.flat();
+    const dumpers = annotationFormats.dumpers.flat();
```

Comment:
Probably flat() doesn't needed here anymore. Loaders and dumpers are expected to be flattened.
Just ``const { loaders, dumpers } = annotationFormats;``